### PR TITLE
Add ts back to human log lines.

### LINKED ISSUE #3711: Add timestamps back to human log format
For a human, these are valuable, but as a machine, idgaf because I'm stamping my own.

### PR DESCRIPTION
**Pull request by JTCunning**
@mattrobenolt 

Fixes #3711

---

This change is [Reviewable](https://reviewable.io/reviews/getsentry/sentry/3713)

### CODE REVIEW

File: src/sentry/runner/commands/devserver.py [hunk 1]
```diff
@@ -72,8 +72,8 @@ def devserver(reload, watchers, workers, bind):
             raise click.ClickException('Disable CELERY_ALWAYS_EAGER in your settings file to spawn workers.')
 
         daemons += [
-            ('worker', ['sentry', 'run', 'worker', '-c', '1', '-l', 'INFO', '--autoreload']),
-            ('cron', ['sentry', 'run', 'cron', '-l', 'INFO', '--autoreload']),
+            ('worker', ['sentry', 'run', 'worker', '-c', '1', '--autoreload']),
```
Comment (mattrobenolt) [inline src/sentry/runner/commands/devserver.py]: Why you remove these? The default level is `WARN` for celery.

Comment (JTCunning) [inline src/sentry/runner/commands/devserver.py]: This now overrides sentry setting the log level, so if I do `devserver -l warning` celery will still log in `INFO`.

Comment (mattrobenolt) [inline src/sentry/runner/commands/devserver.py]: Ah, that's fair.

**Comment by mattrobenolt** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/getsentry/sentry/3713)*

**Pull request merged**